grommet-leaflet-core/src/markers/Cluster.jsx

```diff
@@ -44,6 +38,7 @@ const getClusterStatus = childMarkers => {
     if (status === 'critical') mostSevere = 'critical';
     else if (status === 'warning') mostSevere = 'warning';
     else if (status === 'unknown') mostSevere = 'unknown';
+    else if (status === 'good') mostSevere = 'good';
```

Comment:
Should a Default marker/cluster and Status marker/cluster be their own distinct things? e.g. Pin and Cluster get the default treatment, whereas StatusPin and StatusCluster are an explicit flavor of pin and cluster. Caller would be specifically opting into a status or non-status context.

Reply:
Yeah that makes sense, I'll play around with heading in this direction

Reply:
I explored separating Pin into a regular Pin and a StatusPin, I also explored going the opposite direction and having one Pin component and having different 'kinds' of Pin coming from the theme.

I felt that having one Pin component and having different kinds of pins defined in the theme felt the most natural to me. A status Pin doesn't feel like a distinct component but rather a kind of pin that could be displayed. The 'kind' pattern allows for flexibility on the Grommet community side, people can define any kind of pin they need and from the hpe side we can have the status pin kinds defined in the leaflet hpe theme.

Taylor and I chatted about this earlier today and felt we were both on the same page but let me know if there is any concern with this direction.